edgelet/api/managementVersion_2019_01_30.yaml

```diff
@@ -436,7 +436,24 @@ paths:
         default:
           description: Error
           schema:
-            $ref: '#/definitions/ErrorResponse'            
+            $ref: '#/definitions/ErrorResponse'
+
+  '/device/reprovision':
```

Comment:
Because this is adding a new API, I think there should be a bump on the version.

Reply:
Bumped the version now